packages/xchain-cosmos/src/util.ts

```diff
@@ -126,7 +132,7 @@ export const getTxsFromHistory = (txs: TxResponse[], mainAsset: Asset): Tx[] =>
 
           if (from_index === -1) {
             from.push({
-              from: input.address,
+              from: input.address || '',
```

Comment:
What's the case to have an empty address for `from`?

Reply:
cosmos.bank.v1beta1.IInput.address?: string | null | undefined